app/mailers/publisher_notifications.rb

```diff
@@ -0,0 +1,16 @@
+class PublisherNotifications < Mail::Notify::Mailer
+  include ApplicationHelper
+
+  TEMPLATE_ID = ENV.fetch("GOVUK_NOTIFY_TEMPLATE_ID", "test-template-id").freeze
```

Comment:
We've implemented this differently in other places that use Notify (as a method on the base class) but I'm not sure it's important enough to change.